types/qr-image/index.d.ts

```diff
@@ -13,12 +13,34 @@ export type ec_level = 'L' | 'M' | 'Q' | 'H';
 /** @default 'png' */
 export type image_type = 'png' | 'svg' | 'pdf' | 'eps';
 
+/**
+ * bitmap.size — width (and height) of resulting image in pixels
```

Comment:
Extremely minor quibble, but this would probably be more accessible if split up and put on `size` and `data` individually.

Reply:
not worries, this was taken 'as is' from author's sample code. So now I've just refined into proper JSDoc comments as requested in the update. Thx!